.changeset/neat-mangos-judge.md

```diff
@@ -0,0 +1,6 @@
+---
+'astro': minor
+---
+
+#9092 - Fallback to `tap` prefetch strategy on slow connection, instead of bailing.
+#9092 - Add `ignoreSlowConnection` option to prefetch API, which lets manually prefetch even on data saver mode or slow connection.
```

Comment:
I think you can create two changesets for this. We also usually follow a present tense here, so e.g. this can be splitted as:

```
Fallback to the `tap` prefetch strategy on slow connection, instead of bailing.
```

```
Adds a `ignoreSlowConnection` option to the `prefetch()` API to prefetch even on data saver mode or slow connection.
```

The PR number here is also not needed.